packages/twenty-server/src/engine/core-modules/function/function.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+
+import { FunctionService } from 'src/engine/core-modules/function/function.service';
+import { FunctionResolver } from 'src/engine/core-modules/function/function.resolver';
+import { UserModule } from 'src/engine/core-modules/user/user.module';
+import { TwentyORMModule } from 'src/engine/twenty-orm/twenty-orm.module';
+import { FunctionWorkspaceEntity } from 'src/modules/function/standard-objects/function.workspace-entity';
+import { CustomCodeEngineModule } from 'src/engine/integrations/custom-code-engine/custom-code-engine.module';
+
+@Module({
+  imports: [
+    TwentyORMModule.forFeature([FunctionWorkspaceEntity]),
+    CustomCodeEngineModule,
+    UserModule,
+  ],
+  providers: [FunctionService, FunctionResolver],
+})
+export class FunctionModule {}
```

Comment:
I think we need to organize the code in a more modular way, otherwise we are going to run into circular dependencies. I would propose:
- within engine/metadata-modules: FunctionMetadataModule. This is what we will use to update metadata (resolvers, service) and trigger the build logic
- within engine/core-modules: CodeBuilderModule and CodeExecutorModule (or one that would regroup them all)

Reply:
ok